ignite/services/chain/init.go

```diff
@@ -36,17 +36,12 @@ func (c *Chain) Init(ctx context.Context, initAccounts bool) error {
 }
 
 // InitChain initializes the chain.
-func (c *Chain) InitChain(ctx context.Context) error {
+func (c *Chain) InitChain(ctx context.Context, conf *chainconfig.Config, updatePlugin bool) error {
```

Comment:
I would rather call the variable `updateNodeConfiguration` to be more explicit

Maybe we should even, but not in the PR, remove the plugin things, as it was a model we had when we were supporting `stargate` and `launchpad` SDK version